pyroscope/remote.go

```diff
@@ -171,16 +177,13 @@ func (r *remote) uploadProfile(j *uploadJob) error {
 	defer response.Body.Close()
 
 	// read all the response body
-	_, err = ioutil.ReadAll(response.Body)
+	respBody, err := io.ReadAll(response.Body)
```

Comment:
had to revert this one: https://github.com/pyroscope-io/client/commit/4d229a8e7b412a42e960ce754f4aaf4d3749c7b8

Reply:
Ugh thanks!